test/SUMMARY.md

```diff
@@ -0,0 +1,88 @@
+# Tests Summary
+
+There are 9 solidity files, totalling 939 sloc.
+
+| filename                      | language | code | comment | blank | total |
+| :---------------------------- | :------- | :--- | :------ | :---- | :---- |
+| src/contracts/BCoWConst.sol   | solidity | 4    | 10      | 2     | 16    |
+| src/contracts/BCoWFactory.sol | solidity | 20   | 13      | 7     | 40    |
+| src/contracts/BCoWPool.sol    | solidity | 90   | 41      | 24    | 155   |
+| src/contracts/BConst.sol      | solidity | 23   | 29      | 9     | 61    |
+| src/contracts/BFactory.sol    | solidity | 44   | 17      | 11    | 72    |
+| src/contracts/BMath.sol       | solidity | 128  | 156     | 18    | 302   |
+| src/contracts/BNum.sol        | solidity | 133  | 40      | 28    | 201   |
+| src/contracts/BPool.sol       | solidity | 473  | 104     | 107   | 684   |
+| src/contracts/BToken.sol      | solidity | 24   | 27      | 7     | 58    |
+
+
+## Interdependencies
+BCoWFactory deploys a bcowpool
+BCoWPool is a bpool which adds signature validation
+Factory deploys a pool and can "collect" from the pool
+Pool inherit btoken (which represents a LP) and bmath
+Bmath uses bnum
+
+# Unit tests
+Current coverage is XXX % for the 9 contracts, accross XXX tests. All tests are passing. Unit tests are writtent using the branched-tree technique and Bulloak as templating tool.
```

Comment:
this looks more out of an audit report than being part of the repo imho

Reply:
Why being straight up insulting now?!? Jk, should I remove this? Idea is to have a quick overview of all the tests and how the (should) behave